libavfilter/signature_lookup.c

```diff
@@ -541,6 +542,7 @@ static MatchingInfo lookup_signatures(AVFilterContext *ctx, SignatureContext *sc
     cs2 = second->coarsesiglist;
 
     /* score of bestmatch is 0, if no match is found */
+    memset(&bestmatch, 0x00, sizeof(MatchingInfo));
```

Comment:
Nitpick: why "0x00"?

Reply:
This is because "bestmatch" returns as is, without initializing the "bestmatch.first" field.

Reply:
Yeah, I get the need to zero out the memory. Just curious why write "0x00" where "0x0" or even "0" would do just fine. But feel free to ignore me :-)